configs/ld/ld_gflv1_r101dcn_r101_fpn_coco_2x.py

```diff
@@ -0,0 +1,43 @@
+_base_ = ['./ld_gflv1_r101_r18_fpn_coco_1x.py']
+
+model = dict(
+    pretrained='torchvision://resnet101',
+    teacher_config='configs/gfl/gfl_r101_fpn_dconv_c3-c5_mstrain_2x_coco.py',
+    teacher_ckpt=None,
```

Comment:
Why it is None here? You may just use http source file link as the teacher ckpt.